<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTextEdit" name="textEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_F">
    <property name="title">
     <string>File(&amp;F)</string>
    </property>
    <addaction name="actionNew_project_N"/>
    <addaction name="actionOpen_O"/>
    <addaction name="actionSave_as_S"/>
   </widget>
   <widget class="QMenu" name="menuNew_N">
    <property name="title">
     <string>New(&amp;N)</string>
    </property>
    <addaction name="actionCut_C"/>
    <addaction name="actionCopy_C"/>
    <addaction name="actionPaste_P"/>
   </widget>
   <widget class="QMenu" name="menuOpen_O">
    <property name="title">
     <string>View(&amp;V)</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuBuild_B">
    <property name="title">
     <string>Build(&amp;B)</string>
    </property>
    <addaction name="actionBuild_All_Projects"/>
   </widget>
   <addaction name="menu_F"/>
   <addaction name="menuNew_N"/>
   <addaction name="menuOpen_O"/>
   <addaction name="menuBuild_B"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionNew_project_N">
   <property name="text">
    <string>New project(&amp;N)</string>
   </property>
  </action>
  <action name="actionOpen_O">
   <property name="text">
    <string>Open(&amp;O)</string>
   </property>
  </action>
  <action name="actionSave_as_S">
   <property name="text">
    <string>Save as(&amp;S)</string>
   </property>
  </action>
  <action name="actionCut_C">
   <property name="text">
    <string>Cut(&amp;T)</string>
   </property>
  </action>
  <action name="actionCopy_C">
   <property name="text">
    <string>Copy(&amp;C)</string>
   </property>
  </action>
  <action name="actionPaste_P">
   <property name="text">
    <string>Paste(&amp;P)</string>
   </property>
  </action>
  <action name="actionBuild_All_Projects">
   <property name="text">
    <string>Build All Projects</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
